src/AppInstallerRepositoryCore/Rest/Schema/1_1/Json/ManifestDeserializer_1_1.cpp

```diff
@@ -123,6 +126,27 @@ namespace AppInstaller::Repository::Rest::Schema::V1_1::Json
                 installer.Markets.AllowedMarkets = V1_0::Json::ManifestDeserializer::ConvertToManifestStringArray(
                     JsonHelper::GetRawStringArrayFromJsonNode(marketsNode.value().get(), JsonHelper::GetUtilityString(AllowedMarkets)));
             }
+
+            // Expected return codes
+            std::optional<std::reference_wrapper<const web::json::array>> expectedReturnCodesNode = JsonHelper::GetRawJsonArrayFromJsonNode(installerJsonObject, JsonHelper::GetUtilityString(ExpectedReturnCodes));
+            if (expectedReturnCodesNode)
+            {
+                for (auto& returnCodeNode : expectedReturnCodesNode.value().get())
+                {
+                    ExpectedReturnCodeEnum returnResponse = Manifest::ConvertToExpectedReturnCodeEnum(JsonHelper::GetRawStringValueFromJsonNode(returnCodeNode, JsonHelper::GetUtilityString(ReturnResponse)).value_or(""));
+                    DWORD installerReturnCode = JsonHelper::GetRawIntValueFromJsonNode(returnCodeNode, JsonHelper::GetUtilityString(InstallerReturnCode)).value_or(0);

```

Comment:
>JsonHelper::GetRawIntValueFromJsonNode [](http://example.com/codeflow?start=48&length=38)

Code analysis did not complain about converting from signed to unsigned? I think we need a static cast here